Source/Widgets/InfoBox/InfoBox.css

```diff
@@ -14,22 +14,28 @@
     box-shadow: 0 0 10px 1px #000;
     -webkit-transform: translate(100%, 0);
     -moz-transform: translate(100%, 0);
+    -ms-transform: translate(100%, 0);
+    -o-transform: translate(100%, 0);
     transform: translate(100%, 0);
     visibility: hidden;
     opacity: 0;
     -webkit-transition: visibility 0s 0.2s, opacity 0.2s ease-in, -webkit-transform 0.2s ease-in;
     -moz-transition: visibility 0s 0.2s, opacity 0.2s ease-in, -moz-transform 0.2s ease-in;
+    -o-transition: visibility 0s 0.2s, opacity 0.2s ease-in, transform 0.2s ease-in;
     transition: visibility 0s 0.2s, opacity 0.2s ease-in, transform 0.2s ease-in;
```

Comment:
As with my previous comment, I think it's safer to just leave the prefixes there (for now) rather than try and guess at what is or isn't needed.  Eventually a tool will handle it for us.